### PR TITLE
ref(face-landmarks) refactor namings

### PR DESCRIPTION
**Pull request by quitrk**

### CODE REVIEW

File: conference.js [hunk 1]
```diff
@@ -3116,12 +3116,12 @@ export default {
...
     /**
-     * Sends a facial expression as a string and its duration as a number
-     * @param {object} payload - Object containing the {string} facialExpression
+     * Sends a face expression as a string and its duration as a number
+     * @param {object} payload - Object containing the {string} faceExpression
      * and {number} duration
      */
-    sendFacialExpression(payload) {
-        room.sendFacialExpression(payload);
+    sendFaceExpression(payload) {
```
Comment (saghul) [inline conference.js:3123]: Where is this function used? This is the legacy conference object and I everything else face-landmark related is built with React, I don't see this used anywhere. Am I missing something?

File: react/features/face-landmarks/actionTypes.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+export const ADD_FACE_EXPRESSION = 'ADD_FACE_EXPRESSION';
+
+/**
+ * Redux action type dispatched in order to set recognition active in the state.
+ *
+ * {
+ *      type: START_FACE_LANDMARKS_DETECTION
+ * }
+ */
+export const START_FACE_LANDMARKS_DETECTION = 'START_FACE_LANDMARKS_DETECTION';
```
Comment (saghul) [inline react/features/face-landmarks/actionTypes.js:21]: style: alpha sort

File: react/features/face-landmarks/functions.js [hunk 3]
```diff
@@ -60,44 +60,44 @@ export function sendFaceBoxToParticipants(
...
+export function sendFaceExpressionToServer(
         conference: Object,
-        facialExpression: string,
+        faceExpression: string,
         duration: number
 ): void {
     try {
-        conference.sendFacialExpression({
-            facialExpression,
+        conference.sendFaceExpression({
```
Comment (saghul) [inline react/features/face-landmarks/functions.js:76]: Didn't you rename this to sendFaceLandmark?

File: react/features/face-landmarks/functions.js [hunk 4]
```diff
@@ -110,7 +110,7 @@ export async function sendFacialExpressionsWebhook(state: Object) {
         meetingFqn: extractFqnFromPath(),
         sessionId: conference.sessionId,
         submitted: Date.now(),
-        emotions: facialExpressionsBuffer,
+        emotions: faceExpressionsBuffer,
```
Comment (saghul) [inline react/features/face-landmarks/functions.js:113]: Can we consolidate in "expressions" here?

File: react/features/face-landmarks/reducer.js [hunk 5]
```diff
@@ -22,36 +22,36 @@ const defaultState = {
...
     recognitionActive: false
 };
 
-ReducerRegistry.register('features/facial-recognition', (state = defaultState, action) => {
+ReducerRegistry.register('features/face-landmarks', (state = defaultState, action) => {
     switch (action.type) {
-    case ADD_FACIAL_EXPRESSION: {
-        state.facialExpressions[action.facialExpression] += action.duration;
+    case ADD_FACE_EXPRESSION: {
+        state.faceExpressions[action.faceExpression] += action.duration;
```
Comment (saghul) [inline react/features/face-landmarks/reducer.js:32]: There was a bug here before. This mutates the state, it should not.

**Review by saghul — CHANGES_REQUESTED** [review]
Left a few minor comments, thanks for doing this!

File: conference.js (hunk 1 above)
Comment (quitrk) [reply conference.js:3123]: https://github.com/jitsi/jitsi-meet/pull/11307/files#diff-3df1b963ab402f0e79ffdeaacca3881791fbedc759fdce6d7fb37353c23f57abR76

File: react/features/face-landmarks/functions.js (hunk 3 above)
Comment (quitrk) [reply react/features/face-landmarks/functions.js:76]: no

File: react/features/face-landmarks/functions.js (hunk 4 above)
Comment (quitrk) [reply react/features/face-landmarks/functions.js:113]: looks like a backend change

File: react/features/face-landmarks/functions.js (hunk 3 above)
Comment (quitrk) [reply react/features/face-landmarks/functions.js:76]: actually yes..

File: conference.js (hunk 1 above)
Comment (quitrk) [reply conference.js:3123]: nevermind, this is the store conference

**Comment by quitrk** [comment]
Jenkins please test this please.

**Pull request merged**